generators/app/templates/src/containers/TextEditor/text-editor.component.js

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable constructor-super */
+import React from 'react';
+import { TextEditorWrapper, TextEditorContainer, Header, Form, FullGridSize, Button, Label, Input, TextArea } from './text-editor.style';
+import SolidAuth from 'solid-auth-client';
+import { successToaster, errorToaster } from '@utils';
+
+class Editor extends React.Component {
+  state
+  constructor(props) {
+    super(props);
+    this.state = {url: 'https://example-pod.com/shared/some-doc.txt'};
+
+    this.handleUrlChange = this.handleUrlChange.bind(this);
+    this.handleTextChange = this.handleTextChange.bind(this);
+    this.handleLoad = this.handleLoad.bind(this);
+    this.handleSave = this.handleSave.bind(this);
+  }
+
+  handleUrlChange(event) {
+    this.setState({url: event.target.value});
+  }
+
+  handleTextChange(event) {
+    this.setState({text: event.target.value});
+  }
+
+  handleLoad(event) {
+    console.log('handle load!');
+    event.preventDefault();
+    console.log('default prevented!');
+    const doc = SolidAuth.fetch(this.state.url);
+    doc.then(async (response) => {
+      const text = await response.text();
+      console.log('fetched', text);
+      this.setState({ text });
+    });
+  }
+
+  async handleSave(event) {
+    event.preventDefault();
+    const result = await SolidAuth.fetch(this.state.url, {
```

Comment:
This creates a new file, correct? I don't think that's possible yet in LDFlex, but it may be worth a look, as we're trying to consolidate onto a single tool as much as possible. If it's not possible then this is fine, but we should add a comment for new developers explaining what the intention of this block is

Reply:
ok sure, will rewrite this to use LDFlex.

Reply:
Actually it doesn't make sense to use LDFlex or TripleDoc for a non-RDF resource

Reply:
I added a comment to that effect.